.travis_scripts/create_feedstocks

```diff
@@ -35,9 +35,7 @@ bash ~/miniconda.sh -b -p ~/miniconda
 )
 source ~/miniconda/bin/activate root
 
-conda install --yes --quiet git=2.12.2
-conda install -n root --yes --quiet jinja2 anaconda-client
-conda install --yes --quiet conda-build=2 conda-smithy
+conda install --yes --quiet conda-forge-ci-setup=1.* conda-smithy=3.* conda-forge-pinning git=2.12.2
```

Comment:
why pin git?

Reply:
Some of the scripts here fails with the newer git versions, https://github.com/conda-forge/staged-recipes/issues/3365

Reply:
Yeah, this is a bit annoying, but I don't think it needs to be solved in this PR.

Reply:
This was supposed to be merged after conda-smithy=3 is released. Now feedstock conversion is broken due to this

Reply:
Added a simple reversion in PR ( https://github.com/conda-forge/staged-recipes/pull/5657 ) for now.